client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -155,13 +233,16 @@ export class SimpleSiteRenameForm extends Component {
 						<FormSectionHeading>{ translate( 'Change Site Address' ) }</FormSectionHeading>
 						<FormTextInputWithAffixes
 							type="text"
-							value={ this.state.domainFieldValue }
+							value={ domainFieldValue }
 							suffix={ currentDomainSuffix }
 							onChange={ this.onFieldChange }
 							placeholder={ currentDomainPrefix }
-							isError={ !! domainFieldError }
+							isError={ shouldShowValidationMessage && ! isAvailable }
 						/>
-						{ domainFieldError && <FormInputValidation isError text={ domainFieldError } /> }
+						{ shouldShowValidationMessage &&
+							validationMessage && (
+								<FormInputValidation isError={ ! isAvailable } text={ validationMessage } />
```

Comment:
It's visually jarring to have the validation message pop in and grow the height of the container:

![site-address-validate-message-pop-in](https://user-images.githubusercontent.com/1587282/38772128-960dd7ee-3ffd-11e8-85e6-c2352383bb88.gif)

Can you please always render the `FormInputValidation` & make sure it has a min-height set so it doesn't collapse?

Reply:
For sure, this is a little tricky to really get 'perfect' as there are a couple of unknowns (string length, screen width etc.) I think the solution may be just to add white space to the height of a single line. It'll still change height in cases where the string is long enough to break in to multiple lines but it'd be an improvement for sure.

I'll hit this in a follow up PR :)

Reply:
I'm ok if it works in most cases without the jump (whether that's a min-height of 1 line height or 2).

> I'll hit this in a follow up PR :)

Thanks!